src/test/resources/testDataFiles/LoginUser.json

```diff
@@ -10,5 +10,7 @@
     "AccountCreated": "ACCOUNT CREATED!",
     "AccountDeleted": "ACCOUNT DELETED!",
     "Login": "Login to your account"
-  }
+  },
+  "UserEMail" : "abdelazizaliabdelaziz@gmail.com",
+  "UserPassWD" : "karimAhmed"
```

Comment:
- We agreed not to use our names in any of the test data.
- Also, please make sure to create a new test data json file for your test class, and not to use other test data json file for other test class

Reply:
Done